about.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>js Solutions | About Page</title>
+    <!-- css -->
+    <link rel="stylesheet" href="styles.css"/>
+     <!-- font aweosome -->
+     <script src="https://kit.fontawesome.com/d7195d9855.js" crossorigin="anonymous"></script>
+</head>
+<body id="body">
+    <!-- wrapper -->
+    <main class="wrapper" id="wrapper">
+        <!-- header -->
+        <header class="topbar">
+            <div></div>
+            <ul class="links-box">
+                <li class="link"><a class="link-a" href="https://web.facebook.com/humza74/" target="blank"><i class="fa-brands fa-facebook-f fa-xl"></i></a></li>
+                <li class="link"><a class="link-a" href="https://twitter.com/anwarhamza919/" target="blank"><i class="fa-brands fa-twitter fa-xl"></i></a></li>
+                <li class="link">English</li>
+                <li class="link">My Page</li>
+                <li class="link">Logout</li>
+            </ul>
+        </header>
+        <!-- end of header -->
+
+        <!-- navbar -->
+        <section class="navbox" id="navbox">
+            <div class="bars" id="bars"><i class="fa-solid fa-bars fa-xl"></i></div>
+            <div class="cross" id="cross"><i class="fa-solid fa-xmark fa-xl"></i></div>
+            <a class="logo-link" id="logo-link" href="/index.html">
+                <i class="fa-solid fa-code fa-xl"></i>
+                <span class="logo-text">js Solutions</span>
+            </a>
+            <nav class="items-box" id="items-box">
+                <span class="items"><a class="items-a active" href="/about.html">About</a></span>
+                <span class="items"><a class="items-a" href="#">Program</a></span>
+                <span class="items"><a class="items-a" href="#">Join</a></span>
+                <span class="items"><a class="items-a" href="#">Sponsor</a></span>
+                <span class="items"><a class="items-a" href="#">News</a></span>
+                <button class="cc-campaign" id="campaign-btn" type="button">CC Campaign</button>
+            </nav>
+        </section>
+        <!-- end of navbar -->
+
+        <!-- hero -->
+        <section class="hero-box about">
+            <h1 class="headline about">GET YOUR JS SOLUTIONS <br>2022</h1>
```

Comment:
- [ ]  Kindly change this h1 tag h2 tag, It makes your website easier to read and more semantic by screen readers 👍🏿

Reply:
but this is the only h1 tag I used on the entire page, I would really appreciate it if you elaborate your point